src/app/globals.css

```
@@ -1,26 +1,38 @@
 :root {
-  --background: #ffffff;
-  --foreground: #171717;
+  --background: #f7f3ea;
+  --foreground: #172033;
+  --surface: rgba(255, 255, 255, 0.72);
+  --surface-border: rgba(23, 32, 51, 0.12);
+  --accent: #d95d39;
+  --accent-strong: #1b5e6f;
+  --shadow: 0 30px 80px rgba(18, 25, 40, 0.18);
+  font-family: Inter, ui-sans-serif, system-ui, -apple-system, BlinkMacSystemFont,
+    'Segoe UI', sans-serif;
```

Stylelint (17.6.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)